src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -341,6 +343,7 @@ def load_arguments(self, _):
         c.argument('nat_gateway_managed_outbound_ip_count', type=int, validator=validate_nat_gateway_managed_outbound_ip_count)
         c.argument('nat_gateway_idle_timeout', type=int, validator=validate_nat_gateway_idle_timeout)
         c.argument('network_dataplane', arg_type=get_enum_type(network_dataplanes))
+        c.argument('network_policy')
```

Comment:
Is the value of this param enumerable? If so, please add `arg_type=get_enum_type` to the param.

Reply:
We want to avoid referencing 'none' (that was introduced in API version 2023-08-02-preview) until it is ready. See comment: https://github.com/Azure/azure-cli-extensions/pull/6809#discussion_r1337806199